docs/sources/mimir/set-up/jsonnet/configure-autoscaling.md

```diff
@@ -17,11 +17,22 @@ Mimir Jsonnet supports autoscaling for the following components:
 - [Querier]({{< relref "../../references/architecture/components/querier" >}})
 - [Distributor]({{< relref "../../references/architecture/components/distributor" >}})
 
-Autoscaling, which is based on Prometheus metrics and [Kubernetes-based Event Driven Autoscaler (KEDA)](https://keda.sh), uses Kubernetes’ Horizontal Pod Autoscaler (HPA).
+Autoscaling, which is based on Prometheus metrics and [Kubernetes-based Event Driven Autoscaler (KEDA)](https://keda.sh), uses Kubernetes' Horizontal Pod Autoscaler (HPA).
 
 HPA is not configured directly in Jsonnet but it's created and updated by KEDA.
 KEDA is an operator, running in the Kubernetes cluster, which is responsible to simplify the setup of HPA with custom metrics (Prometheus in our case).
 
+{{< admonition type="warning" >}}
+Do not use the same Mimir or Grafana Enterprise Metrics cluster for storing and querying autoscaling metrics. Using the same cluster can create a dangerous feedback loop:
+
+1. If the Mimir/GEM cluster becomes unavailable, autoscaling will stop working because it cannot query the metrics
+2. This can prevent the cluster from automatically scaling up during high load or recovery
+3. The inability to scale can further exacerbate the cluster's unavailability
+4. This may prevent the cluster from recovering
```

Comment:
```suggestion
Don't use the same Mimir or Grafana Enterprise Metrics cluster for storing and querying autoscaling metrics. Using the same cluster can create a dangerous feedback loop.

For instance, if the Mimir or GEM cluster becomes unavailable, autoscaling stops working, because it cannot query the metrics. This prevents the cluster from automatically scaling up during high load or recovery. This inability to scale further exacerbates the cluster's unavailability, which might, in turn, prevent the cluster from recovering.

Instead, use a separate Prometheus instance or a different metrics backend for autoscaling metrics.
```